.github/workflows/jit-format.yml

```diff
@@ -45,7 +50,10 @@ jobs:
       
       - name: Run jitformat.py
         run: |
-          python3 runtime/src/coreclr/scripts/jitformat.py --jitutils jitutils -r ${{ github.workspace }}/runtime -o ${{ matrix.os.name }} -a x64 ${{ matrix.os.cross }}
+          cd runtime
+          python3 src/coreclr/scripts/jitformat.py --jitutils $(pwd)/../jitutils -r $(pwd) -o ${{ matrix.os.name }} -a x64 ${{ matrix.os.cross }}
```

Comment:
nit:

```yaml
- name: Run jitformat.py
  working-directory: runtime
  run: python3 src/coreclr/scripts/jitformat.py --jitutils ../jitutils -r $(pwd) -o ${{ matrix.os.name }} -a x64 ${{ matrix.os.cross }}
```
and dotnet to v4 as well (https://github.com/actions/setup-dotnet?tab=readme-ov-file#usage)